MinecraftClient/Resources/lang/ru.ini

```
@@ -498,4 +498,4 @@ bot.scriptScheduler.task=triggeronfirstlogin: {0}\n triggeronlogin: {1}\n trigge
 
 # TestBot
 bot.testBot.told=Bot: {0} сказал мне : {1}
-bot.testBot.said=Бот: {0} сказал : {1}
\ No newline at end of file
+bot.testBot.said=Бот: {0} сказал : {1}
```

Should the space here one also be removed?
```suggestion
bot.testBot.said=Бот: {0} сказал: {1}
```

---

Yes

---

OK! Applied suggestion and merged PR 👍